src/components/Modals/NewCourse/CourseConflictModal.vue

```diff
@@ -201,7 +275,45 @@ export default defineComponent({
         }
       });
 
-      return [...conflictReqIds, ...selectableReqIdsInConflict];
+      relatedReqIds.forEach(relatedReqId => {
+        const constraintViolations = getConstraintViolationsForSingleCourse(
+          { uniqueId: uniqueID },
+          [...conflictReqIds, relatedReqId],
```

Comment:
This code got a bit overcomplicated 😕. As far as I can tell, we currently store requirements that are in conflict on a course, and a list of "related" requirements that a course can apply to. Thus, when editing requirements, I have to check the related requirements to generate the list of requirements that are not currently in a course conflict, but should show up unchecked as it was an option when the course was added.